<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VehicleInfoWidget</class>
 <widget class="QWidget" name="VehicleInfoWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>323</width>
    <height>390</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTextBrowser" name="textBrowser_oneVehicle">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>291</width>
     <height>211</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>所有车辆信息</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>选择车辆编号</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_allVehicle">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>291</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_vehicleSelect">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>130</y>
     <width>191</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
